DerekNgu/lab1_hard_project/src/main.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+#include <hello_world.h>
+
+int main()
+{
+    std::cout << "Hello, World!";
+    hello_world::printPartTwo();
```

Comment:
This is calling a static function, you should instead create an instance of your hello world class instead of directly calling a static function that isn't part of an object instance.

Reply:
https://cplusplus.com/doc/tutorial/classes/